src/xwayland/XWayland.hpp

```diff
@@ -17,18 +17,22 @@ class CXWM;
 
 class CXWayland {
   public:
-    CXWayland(const bool enabled);
+    CXWayland(const bool wantsEnabled);
 
 #ifndef NO_XWAYLAND
     std::unique_ptr<CXWaylandServer> pServer;
     std::unique_ptr<CXWM>            pWM;
 #endif
+    bool enabled();
 
     void setCursor(unsigned char* pixData, uint32_t stride, const Vector2D& size, const Vector2D& hotspot);
 
     struct {
         CSignal newSurface;
     } events;
+
+  private:
+    bool m_enabled;
```

Comment:
still not initializeddd

Reply:
Sorry, I don't write c++ as my day job :sweat_smile: Hopefully good this time